src/gui/MainWindow.ui

```diff
@@ -271,6 +271,7 @@
     <addaction name="menuPython"/>
     <addaction name="separator"/>
     <addaction name="fileShowLibraryFolder"/>
+    <addaction name="fileShowTreasureTrove"/>
```

Comment:
Perhaps name this smth. like "Open File from Backup" and put it higher up in the file menu?